packages/did/src/DidDetails/LightDidDetails.ts

```diff
@@ -6,20 +6,50 @@
  */
 
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable import/prefer-default-export */
 
 import { SDKErrors, Crypto } from '@kiltprotocol/utils'
 import { encodeAddress } from '@polkadot/util-crypto'
 import {
   getEncodingForSigningKeyType,
   getKiltDidFromIdentifier,
+  parseDidUrl,
 } from '../Did.utils'
-import type { INewPublicKey } from '../types'
+import type { LightDidDetailsCreationOpts } from '../types'
 import { DidDetails } from './DidDetails'
 import { serializeAndEncodeAdditionalLightDidDetails } from './LightDidDetails.utils'
 
-export interface LightDidDetailsCreationOpts {
-  authenticationKey: INewPublicKey
-  encryptionKey?: INewPublicKey
+function checkLightDidCreationOptions(
```

Comment:
It might be nicer to remove `checkLightDidCreationOptions` inside the `utils`.

Reply:
https://github.com/KILTprotocol/sdk-js/pull/428/commits/321c32f70411b167465f2bac41cdc3899ac010f6